example/src/main/java/com/spruceid/mobilesdkexample/wallet/GenericCredentialListItem.kt

```diff
@@ -159,18 +160,14 @@ fun GenericCredentialDetailsItem(credentialPack: CredentialPack) {
                     "credentialSubject.driversLicense.family_name",
                     "credentialSubject.driversLicense.birth_date",
                 ),
-                formatter = {values ->
-                    val w3cvc = values.toList()
-                        .first { (key, _) ->
-                            val credential = credentialPack.getCredentialById(key)
-                            (credential?.asJwtVc() != null || credential?.asJsonVc() != null)
-                        }.second
+                formatter = { values ->
+                    val w3cvc = values.toList().first().second
 
                     var portrait = ""
                     var firstName = ""
                     var lastName = ""
                     var birthDate = ""
-                    if(w3cvc["credentialSubject.driversLicense"].toString().isNotEmpty()) {
+                    if (w3cvc["credentialSubject.driversLicense"].toString().isNotEmpty()) {
```

Comment:
Does the `keyPathFinder` function need to be used here instead of indexing?

Reply:
The `keyPathFinder` was incorporated when searching for claims. So, if I have an object in the format

```json
{
  "issuer": {
    "address": {
      "number": 123
    }
  }
}
```
I'll ask for the `issuer.address.number` property, and the formatter returns the object

```json
{
  "issuer.address.number": 123
}
```

That's why I can do index access this way